workshop/controllers/core/clipboard.lua

```diff
@@ -3,13 +3,33 @@ local history = require("tevgit:workshop/controllers/core/history.lua")
 local selection = require("tevgit:workshop/controllers/core/selection.lua")
 
 local clipboard = {}
+local cut = false
 
 keybinder:bind({
     name = "copy",
     key = enums.key.c,
     priorKey = enums.key.leftCtrl,
-	action = function()
-		clipboard = selection.selection
+    action = function()
+        history.beginAction(workspace, "Copy")
```

Comment:
You don’t need to begin action for copying; this will not achieve anything

Reply:
Alright.